builtins/amp-ad.md

```diff
@@ -62,6 +62,47 @@ Most ad networks require further configuration. This can be passed to the networ
 
 Optional attribute to pass configuration to the ad as an arbitrarily complex JSON object. The object is passed to the ad as-is with no mangling done on the names.
 
+#### Ad Resizing
+
+An `amp-ad` must have static layout defined as is the case with any other AMP element. However,
+it's possible to resize an `amp-ad` in runtime. To do so:
+
+1. The `amp-ad` must be defined with `resizable` attribute;
```

Comment:
Probably a good idea actually. Make sure to also have the validator changes done.